media/jsonb/common/src/main/java/io/helidon/media/jsonb/common/JsonbBodyWriter.java

```diff
@@ -48,6 +49,13 @@ private JsonbBodyWriter(Jsonb jsonb) {
     public boolean accept(GenericType<?> type,
             MessageBodyWriterContext context) {
 
+        // We are excluding the following types from support:
+        // 1. any char sequence
+        // 2. Flow.Publisher - that can only be supported by streaming media
+        if (Flow.Publisher.class.isAssignableFrom(type.rawType())) {
```

Comment:
Yes, because otherwise when you have JSON-B registered, you do not get an error, if you do  `response.send(Publisher<Pojo>)`
You should get appropriate error telling you there is no registered writer, but JSON-B just processes it

Reply:
I will look for the place we check for `byte[]` and update it